.gitignore

```diff
@@ -1,7 +1,5 @@
 *.bak
 jplc
-*/out.txt
-*/out.txt2
-*/out.pp
 */expected.pp
 old/
+*/*out*
```

Comment:
This is a bad idea—lots of *words* contain "out" as a substring, and this might accidentally ignore totally reasonable names. Can we give consistent prefixes or postfixes to these files, to avoid problems?

Reply:
This can include renaming the existing `out.txt`/`out.txt2`/`out.pp` files!